src/v1/language_service_client.js

```diff
@@ -394,6 +395,54 @@ class LanguageServiceClient {
   }
 
   /**
+   * Classifies a document into categories.
+   *
+   * @param {Object} request
+   *   The request object that will be sent.
+   * @param {Object} request.document
+   *   Input document.
+   *
+   *   This object should have the same structure as [Document]{@link google.cloud.language.v1.Document}
+   * @param {Object} [options]
+   *   Optional parameters. You can override the default settings for this call, e.g, timeout,
+   *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/global.html#CallOptions} for the details.
+   * @param {function(?Error, ?Object)} [callback]
+   *   The function which will be called with the result of the API call.
+   *
+   *   The second parameter to the callback is an object representing [ClassifyTextResponse]{@link google.cloud.language.v1.ClassifyTextResponse}.
+   * @returns {Promise} - The promise which resolves to an array.
+   *   The first element of the array is an object representing [ClassifyTextResponse]{@link google.cloud.language.v1.ClassifyTextResponse}.
+   *   The promise has a method named "cancel" which cancels the ongoing API call.
+   *
+   * @example
+   *
+   * const language = require('@google-cloud/language');
+   *
+   * var client = new language.v1.LanguageServiceClient({
+   *   // optional auth parameters.
+   * });
+   *
+   * var document = {};
+   * client.classifyText({document: document})
```

Comment:
Just a nitpick, so feel free to ignore but this can be `client.classifyText({document})`